src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.addAlive(this);
+  }
+
+  static alive = [];
+
+  static addAlive(animal) {
+    Animal.alive.push(animal);
```

Comment:
Ensure that `Animal.addAlive(this)` is called in every subclass constructor to maintain the list of alive animals accurately.